Public/Src/Cache/ContentStore/Grpc/ContentStore.proto

```diff
@@ -196,6 +198,19 @@ enum CopyCompression {
     gzip = 1;
 }
 
+// PushFile
+message PushFileRequest {

```

Comment:
>PushFileRequest [](start = 8, length = 15)

Not happy that we are adding extra data to every chunk. Stuff transmitted just once should go in header.